low-level/framework/base/conf_upgrade.py

```diff
@@ -13,12 +13,10 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
```

Comment:
We need to rename this class to Upgrade, so we can use same for other Upgrade related requirements in future

And current class member functions can be renamed as follows
get_changed_keys() -> conf_changed_keys()
create_merged_config() -> conf_merge()
upgrade_existing_config() -> conf_update()
remove_merged_config() -> conf_cleanup()